src/controllers/objecting.entity.name.controller.ts

```diff
@@ -16,6 +22,58 @@ const validators = [
   check(DIVULGE_INFO_FIELD).not().isEmpty().withMessage(ErrorMessages.SELECT_TO_DIVULGE),
 ];
 
+const getExistingPageData = async (session: Session, res: Response, next: NextFunction) => {
+  try {
+    const objection: Objection = await getObjection(session) as Objection;
+    const existingName: string = objection.created_by.fullName;
+    const existingShareIdneity: string = (objection.created_by.shareIdentity === true) ? "yes" : "no";
+    if (existingName && existingShareIdneity) {
+      return res.render(Templates.OBJECTING_ENTITY_NAME, {
+        fullNameValue: existingName,
+        isYesChecked: existingShareIdneity === "yes",
+        isNoChecked: existingShareIdneity === "no",
+        templateName: Templates.OBJECTING_ENTITY_NAME,
+      });
+    } else {
+      return next(new Error("Existing data not present"));
+    }
+  } catch (e) {
+    logger.error(e.message);
+    return next(e);
+  }
+}
+
+const isChangeAnswerFlagPresentSetAndSetToTrue = (session: Session): boolean => {
+  const changeAnswer: boolean = retrieveFromObjectionSession(session, CHANGE_ANSWER_KEY);
+  if (changeAnswer) {
+    return changeAnswer === true;
```

Comment:
change answer is a boolean, can we just return changeAnswer? as retrieveFromObjectionSession should either return a boolean or error.

Reply:
Removed function just use the function call